ci/autoscaler/scripts/register-broker.sh

```diff
@@ -3,7 +3,6 @@
 set -euo pipefail
 
 system_domain="${SYSTEM_DOMAIN:-autoscaler.ci.cloudfoundry.org}"
-service_name="${SERVICE_NAME:-autoscaler}"
```

Comment:
should stay

Reply:
the idea is to use the deployment_name as service_name and service_offering. Deployment name, service name and service offering would match always.